packages/itwin/grouping-mapping-widget/src/widget/components/SelectProject.tsx

```diff
@@ -65,6 +65,7 @@ const SelectProject = ({ onSelect, onCancel }: SelectProjectProps) => {
   }, [getAccessToken]);
 
   const startSearch = useCallback(() => {
+    if(!searchInput) return;
```

Comment:
This change looks unrelated.

Reply:
It is unrelated. Small one line bug that stops a search from being made when no input is given.

Reply:
Formatting inconsistency - there should be a space between `if` and `(`.

Reply:
done